pkg/operator/ceph/cluster/controller.go

```diff
@@ -176,6 +178,20 @@ func (c *ClusterController) StartWatch(namespace string, stopCh chan struct{}) {
 	} else {
 		logger.Infof("Disabling hotplug orchestration via %s", disableHotplugEnv)
 	}
+
+	// watch for "rook-ceph-operator-config" ConfigMap
+	_, rookCephOperatorConfigController := cache.NewInformer(cache.NewFilteredListWatchFromClient(c.context.Clientset.CoreV1().RESTClient(),
+		"configmaps", operatorNamespace, func(options *metav1.ListOptions) {
+			options.LabelSelector = fmt.Sprintf("%s=%s", k8sutil.AppAttr, operatorName)
```

Comment:
We only load settings from a configmap with the name `rook-ceph-operator-config`, right? Instead of watching the label, seems like we should be watching for changes to exactly the configmap with that name. Otherwise, we could get events for additional configmaps with the same label.

Reply:
Completely forgot. Fixed it.